src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.AlreadyExistException;
+import core.basesyntax.exception.InvalidValueException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService service = new RegistrationServiceImpl();
+    private User userOk = new User("vasyl.ivaniv", "1955135", 33);
+    private User user1 = new User("kkooll99", "19941993", 29);
+    private User user1Same = new User("kkooll99", "19941993", 29);
+    private User userWithSmallerLogin = new User("kkool", "19941993", 29);
+    private User userWithSmallerPassword = new User("kkooll98", "1994", 29);
+    private User userIsTooYoung = new User("kkooll20", "19941993", 17);
```

Comment:
One User is good enough
Init it using BeforeEach
You can set one invalid field to it inside test methods where you need it.
```suggestion
```